app/services/platform_apis/properties/client.rb

```diff
@@ -28,7 +28,7 @@ def connection
             url:
               Rails.application.credentials.platform_apis[:properties][:url],
             key:
-              Rails.application.credentials.platform_apis[:properties][:x_api_key]
+              Rails.application.credentials.platform_apis[:properties][:api_key]
```

Comment:
Hey I just wonder if all `key` references should be renamed to `token` as that is what JWT is. I know that's a bit annoying lol sorry!  😁